pkg/sql/logictest/testdata/logic_test/serializable_eager_restart

```diff
@@ -33,9 +36,13 @@ user root
 statement ok
 INSERT INTO t(a) VALUES (1)
 
-# Commit will send an EndTransaction which will result in a retry error.
+# RELEASE SAVEPOINT.
```

Comment:
change to `# RELEASE will send an EndTransaction which will result in a retriable error.`

Reply:
Done.